dotnet/src/Experimental/Agents/IChatMessage.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.ObjectModel;
+
+namespace Microsoft.SemanticKernel.Experimental.Agents;
+
+/// <summary>
+/// Represents a message that is part of an agent thread.
+/// </summary>
+public interface IChatMessage
```

Comment:
I wouldn't normally nit on naming...but since naming is the essence of this change, I wonder about `AgentMessage`